gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/oneconvergence_service_node_driver.py

```diff
@@ -160,10 +160,10 @@ def setup_stitching(self, context, admin_context, service_type,
             stitching_port_ip = context.core_plugin.get_port(
                     admin_context,
                     stitching_port_id)['fixed_ips'][0]['ip_address']
-        if context.is_consumer_external:
-            self._add_extra_route(
-                context, stitching_port_ip, stitching_subnet_id,
-                provider['subnets'][0])
+        #if context.is_consumer_external:
```

Comment:
Please remove the commented code. You can add comment specifying why extra_route neds to be supported for E-W as well